.github/workflows/backport.yml

```diff
@@ -19,11 +19,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+
+      - name: Load branch name
+        id: get-branch
+        run: |
+          OLD_BRANCH=$(cat .github/BACKPORT_BRANCH)
+          echo "OLD_BRANCH=${OLD_BRANCH}" >> $GITHUB_ENV
+
       - name: Create backport pull requests
         uses: korthout/backport-action@v3
         with:
           copy_assignees: true
           copy_labels_pattern: true
           copy_requested_reviewers: true
           label_pattern: to-be-backported
-          target_branches: 11.8.x
+          target_branches: ${{ fromJSON(env.OLD_BRANCH) }}
```

Comment:
This is a bug: `fromJSON` would convert strings to numbers and `11.8.x` is not a valid number. Fixing it in #384.